Tools/setup/ubuntu.sh

```diff
@@ -1,5 +1,7 @@
 #! /usr/bin/env bash
 
+set -e
```

Comment:
Maybe a comment why this is useful might save some people some time...

Reply:
https://www.google.com/search?&q=%22set+-e%22 :trollface: